frontend/public/components/monitoring.tsx

```diff
@@ -1261,14 +1261,14 @@ class SilenceForm_ extends React.Component<SilenceFormProps, SilenceFormState> {
                   </label>
                 </div>
                 <div className="col-xs-1">
-                  <button
+                  <Button
                     type="button"
-                    className="btn btn-link btn-link--inherit-color"
                     onClick={() => this.removeMatcher(i)}
                     aria-label="Remove matcher"
+                    variant="plain"
```

Comment:
This was a `link` before.  You intentionally changing it to `plain`?

Reply:
This is just the `MinusCircleIcon` without text so those should be given plain instead of link